docs/enhancements/infrastructure-monitoring-service-api/alarms.md

```diff
@@ -0,0 +1,794 @@
+              "channel": "stable-4.16",
+              "managed_cluster": "89070983-a62f-4dbe-9457-7e0c27832c63",
+              "namespace": "openshift-cluster-version",
+              "openshift_io_alert_source": "platform",
+              "prometheus": "openshift-monitoring/k8s",
+              "severity": "info",
+              "upstream": "<default>"
+          },
+          "annotations": {
+              "description": "For more information refer to 'oc adm upgrade' or https://console-openshift-console.apps.cnfdf27.sno.telco5gran.eng.rdu2.redhat.com/settings/cluster/.",
+              "summary": "Your upstream update recommendation service recommends you update your cluster."
+          },
+          "startsAt": "2024-08-28T11:39:17.958Z",
+          "endsAt": "0001-01-01T00:00:00Z",
+          "generatorURL": "https://console-openshift-console.apps.cnfdf27.sno.telco5gran.eng.rdu2.redhat.com/monitoring/graph?g0.expr=sum+by+%28channel%2C+namespace%2C+upstream%29+%28cluster_version_available_updates%29+%3E+0&g0.tab=1",
+          "fingerprint": "91406cd113ad87e5"
+      },
+ ]
+```
+
+Equivalent AlarmEventRecord in Go 
+
+```go
+alarmEventRecord := AlarmEventRecord{
+		AlarmDefinitionID: "82139b9c-3683-427b-a1f5-4bf9d6dbb0d8", // See section "For a given ResourceTypeID and AlarmName (coming from AM alert), find the AlarmDefinitionID and ProbableCauseID"
+		ProbableCauseID:   "7fe31219-c2a2-48c7-a2f5-f4bcfe0486e7", // See section "For a given ResourceTypeID and AlarmName (coming from AM alert), find the AlarmDefinitionID and ProbableCauseID"
+        AlarmEventRecordID: uuid.New(), // Auto generated with DB                 
+		ResourceTypeID:     "c9d3f9c5-8429-4484-8179-2a7977071bbf", // See section "Notes on Init phase" on how labels.managed_cluster mapped to resourceTypeID
+		ResourceID:        "d43bf16b-c9c6-432b-934e-7b670cc6a2cc", // labels.managed_cluster
+		AlarmRaisedTime:   "12-31-1920 12:32:14Z",                 // comes from alertmanager `startsAt`
+        AlarmAcknowledged: false, 
+        PerceivedSeverity: SeverityCritical,                      // comes from labels.severity
+        Extensions:        []KeyValue{{Key: "namespace", Value: "openshift-cluster-version"}, ....}, // any labels that are not processed already (e.g skip labels.severity)
+
+        // Optionally 
+        AlarmChangedTime         // use changedAt
+        AlarmClearedTime         // use endsAt or potentially current time if we missed the resolved notification
+        AlarmAcknowledgedTime    // Done through API request AlarmEventRecordModifications.alarmAcknowledged (update AlarmAcknowledged)
+}
+```
+
+## Schema
+All O-RAN services will use the same O-CLOUD DB service. More on DB deployment [here](#postgres).
+
+Each table is modeled after O-RAN data structures. DB in our case wil be called `o-ran-infrastructure-monitoring-alarms`.
+Init SQL may look like the following:
+```sql
+CREATE DATABASE o-ran-infrastructure-monitoring-alarms;
```

Comment:
Since we are not sharing this postgres instance there is no need for the database name to be so specific.  Perhaps "alarm-server"? or just "alarms".

Reply:
yep simply `alarms` works. Updated